ijpost/ijpost.php

```diff
@@ -167,17 +169,19 @@ function ijpost_send(&$a,&$b) {
 
 	$tz = 'UTC';
 
-	$x = q("select timezone from user where uid = %d limit 1",
+	$x = q("SELECT `timezone` FROM `user` WHERE `uid`=%d LIMIT 1",
 		intval($b['uid'])
 	);
-	if($x && strlen($x[0]['timezone']))
+
+	if (DBM::is_result($x) && strlen($x[0]['timezone'])) {
```

Comment:
This should be `if (DBM::is_result($x) && !empty($x[0]['timezone'])) {`

Reply:
This is indenting with 4 spaces instead of (more common here) tab. Maybe one day this needs being "fixed".